satellite/console/users_test.go

```diff
@@ -128,6 +149,11 @@ func testUsers(ctx context.Context, t *testing.T, repository console.Users, user
 		assert.Equal(t, userByID.PasswordHash, userByEmail.PasswordHash)
 		assert.Equal(t, userByID.PartnerID, userByEmail.PartnerID)
 		assert.Equal(t, userByID.CreatedAt, userByEmail.CreatedAt)
+		assert.Equal(t, userByID.IsProfessional, userByEmail.IsProfessional)
```

Comment:
This is only testing that the results of GetByEmail() and Get() match - but we probably also want to compare against the original user. See these lines:
```
assert.Equal(t, name, userByEmail.FullName)
assert.Equal(t, lastName, userByEmail.ShortName)
assert.Equal(t, user.PartnerID, userByEmail.PartnerID)
```
below repository.GetByEmail() and repository.Get()

Maybe we can add some asserts below each one like
```
if user.IsProfessional {
    assert.Equal(t, position, userByEmail.Position)
    ...
} else {
    assert.Equal(t, "", userByEmail.Position)
    ...
}
```